src/OctopusPuppet/LogMessager/ILogMessager.cs

```diff
@@ -0,0 +1,19 @@
+﻿using OctopusPuppet.Scheduler;
+
+namespace OctopusPuppet.LogMessager
+{
+    public interface ILogMessager
```

Comment:
Must say that `ILogMessager` is too close in meaning to something that delivers the messages (that's what it is in real life) and there can very well be a component in this code that does have that responsibility.

Maybe just `ILogMessages`, `ILogMessageWordings`, `ICommonDeploymentMessageWordings`? Or maybe you can think of something even better?

Reply:
I was having trouble with a name to call the class. I like ILogMessages

Reply:
I'm also struggling with the name - maybe ILogMessageGenerator or ILogMessageTemplates.

But I'm more wondering why this is an interface? Looks more like a helper/utility class with one and only one implementation.